jscomp/test/test_tagged_template.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var Sql = require("sql").default;
+
+var table = "users";
+
+var id = "5";
+
+var query = Sql([
+      "SELECT * FROM ",
+      " WHERE id = ",
+      ""
+    ], table, id);
```

Comment:
@jfrolich in https://github.com/kevinbarabash/rescript-compiler/pull/2 you suggested that the JS output be changed to use tagged templates.  I'm not familiar with the codegen parts of rescript-compiler.  Where should I be looking to change this output?